src/execution.ts

```diff
@@ -361,6 +367,7 @@ function postProcessResult(
  */
 function compileOperation(context: CompilationContext) {
   const type = getOperationRootType(context.schema, context.operation);
+  const serialExecution = context.operation.operation === "mutation";
```

Comment:
Should we create an enum with operation names?

Reply:
This is the only place where this comparison is made so I do not see the need.